tinyms/data/loader.py

```diff
@@ -110,6 +111,28 @@ def __len__(self):
         return self.size
 
 
+
+
+
+
+
+class BertDataset:
```

Comment:
Please add some annotation for this class.

Reply:
Thanks. I have add some annotation for BertDataset.And I also move/add some codes for suggetions.